install/installer/go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/gitpod-io/gitpod/agent-smith v0.0.0-00010101000000-000000000000
 	github.com/gitpod-io/gitpod/blobserve v0.0.0-00010101000000-000000000000
 	github.com/gitpod-io/gitpod/common-go v0.0.0-00010101000000-000000000000
-	github.com/gitpod-io/gitpod/components/gitpod-db/go v0.0.0-00010101000000-000000000000
+	github.com/gitpod-io/gitpod/components/gitpod-db/go v0.0.0-20221221084930-b19357f31064
```

Comment:
This results in using a dependency on our published repository, rather than a local one. This change shouldn't be here.

Reply:
argh! I unstaged it multiple times already.

Reply:
removed, but as far as I see, regenerating test files fails otherwise.